### PR TITLE
Update turn-on-or-off-preloading-content-for-meetings-using-outlook.md

### PR DESCRIPTION
**Pull request by pupara**
Added support for TPM 2.0

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbOnline/set-up-skype-for-business-online/turn-on-or-off-preloading-content-for-meetings-using-outlook.md [hunk 1]
```diff
@@ -30,39 +30,21 @@ Users can preload content, files, or attachments that are attached to an Outlook
...
- **Check that you are running Windows PowerShell version 3.0 or higher**
-  
-1. To verify that you are running version 3.0 or higher: **Start Menu** > **Windows PowerShell**.
-    
-2. Check the version by typing  _Get-Host_ in the **Windows PowerShell** window.
-    
-3. If you don't have version 3.0 or higher, you need to download and install updates to Windows PowerShell. See [Windows Management Framework 4.0](https://go.microsoft.com/fwlink/?LinkId=716845) to download and update Windows PowerShell to version 4.0. Restart your computer when you are prompted.
-    
-4. You will also need to install the Windows PowerShell module for Skype for Business Online that enables you to create a remote Windows PowerShell session that connects to Skype for Business Online. This module, which is supported only on 64-bit computers, can be downloaded from the Microsoft Download Center at [Windows PowerShell Module for Skype for Business Online](https://go.microsoft.com/fwlink/?LinkId=294688). Restart your computer if you are prompted.
+ > [!Note]
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/set-up-skype-for-business-online/turn-on-or-off-preloading-content-for-meetings-using-outlook.md:35]:
```suggestion
> [!NOTE]
```

File: Skype/SfbOnline/set-up-skype-for-business-online/turn-on-or-off-preloading-content-for-meetings-using-outlook.md [hunk 2]
```diff
@@ -30,39 +30,21 @@ Users can preload content, files, or attachments that are attached to an Outlook
...
+  > Skype for Business Online Connector is currently part of the latest Teams PowerShell module. If you're using the latest Teams PowerShell public release, you don't need to install the Skype for Business Online Connector.
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/set-up-skype-for-business-online/turn-on-or-off-preloading-content-for-meetings-using-outlook.md:36]:
```suggestion
> Skype for Business Online Connector is currently part of the latest Teams PowerShell module. If you're using the latest Teams PowerShell public release, you don't need to install the Skype for Business Online Connector.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**